install_dotfiles.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+#Determine which shell
+SHELL=$(which $SHELL)
+if [[ $SHELL == "/bin/bash" ]]; then
+  SHELL=bash
+fi
+if [[ $SHELL == "/bin/zsh" ]]; then
+  SHELL=zsh
+fi
+
+echo -e "Your shell is $SHELL!"
+
+#Function for installing packages
+installPackage()
+{
+  read -p "$2 Continue and install $1? [y/N] " response
+  if [[ "$response" =~ ^([yY][eE][sS]|[yY])$ ]]; then
+    if ! which "$1"; then
+      echo -e "Installing $1\n....."
+      eval $3
+    else
+      echo -e "$1 is already installed!"
+    fi
+  else
+    echo -e "Skipping installing $1\n..."
+  fi
+}
+
+#Function for installing aliases
+installAliases()
+{
+echo -e "$1 contains the following commands :\n"
+cat $1
+echo -e "\n"
```

Comment:
I don't like this.  But I couldn't find a way to add the new line to the read command on line 36